build.sbt

```diff
@@ -91,6 +93,11 @@ lazy val httpCore = project("akka-http-core")
   .settings(Dependencies.httpCore)
   .settings(VersionGenerator.versionSettings)
   .enablePlugins(BootstrapGenjavadoc)
+  .settings(
+    excludeFilter in (Compile, headerCreate) :=
+      "LastEventId.*" || "ServerSentEvent.*" || "Base64Parsing.scala" || "StringBuilding.scala",
```

Comment:
Hmm, that seems quite brittle to me. The plugin shouldn't touch any files that are not yet copyright Lightbend. Can we do that somehow?

Reply:
It looks like we can match on the previous header text and update accordingly. Will give it a try.

Reply:
Access to existing headers was added by @raboof in https://github.com/sbt/sbt-header/pull/142 and has not yet been released. Since it simplifies a lot I suggest we wait until a new release is out.

Reply:
sbt-header 4.1.0 has been released. I'll update the PR to get rid of the excludes.